add-sitemap-to-redirects-webpack-plugin.js

```diff
@@ -0,0 +1,32 @@
+function AddSitemapToRedirectsWebpackPlugin() {
+  return {
+    apply: (compiler) => {
+      compiler.hooks.emit.tapAsync('AddSitemapToRedirectsWebpackPlugin', emit);
+    },
+  };
+}
+
+function emit(compilation, callback) {
+  const tenant = process.env.SCRIVITO_TENANT;
+  const url = process.env.DEPLOY_PRIME_URL;
+
+  if (url && tenant) {
+    const redirects = compilation.assets['_redirects'].source().toString();
+    const patchedRedirects = redirects.replace(/^\/\* \/.+$/m, rewriteRule(tenant, url));
```

Comment:
This took me very long to figure out. The idea seems to be "insert before the 1st wildcard".
* [ ] Can we make this easier to read?
My proposal is having a placeholder/commented line at the right spot, so we can do a more simple replace.